welcome.js

```diff
@@ -0,0 +1,45 @@
+
+
+class WelcomeModal {
+  constructor(options) {
+    this.languages = options.languages
+    this.onLanguageSelect = options.onLanguageSelect || (() => {})
+
+    this.el = document.createElement('div')
+    this.el.className = 'modal-wrap'
+    this.el.innerHTML = this.render(this.languages)
+    this.langButtons = this.el.querySelectorAll('.language-button')
+
+    Array.prototype.slice.call(this.langButtons).forEach(b => {
+      b.addEventListener('click', evt => {
+        evt.preventDefault()
+        const lang = b.value
+        this.onLanguageSelect(lang)
+        this.close()
+      })
+    })
+  }
+
+  render(languages) {
+
+    const buttons = languages.map(l => `<button class="language-button" value="${l.name}">${l.label}</button>`).join('')
+
+    return `
+      <div class="modal">
+        <h1 class="welcome-message">Welcome!</h1>
```

Comment:
A snappy one-liner/purpose statement would be useful here I think:

```suggestion
        <h1 class="welcome-message">Welcome! Twin Cities Mutual Aid is an up to date resource on mutual aid sites in the Twin Cities.</h1>
```

Reply:
@samanpwbb good call, I just added this!